### DISCUSSION TITLE
Help Posting Questions

### DISCUSSION THREAD

**Discussion by LoganHumphrey** [kcarnold/cs344-exam-23sp#15] [Q&A] [answered]
Hello, 

I found it confusing on how to post our question folder given: 
1. Uploads are disabled 
2. Push properties are denied. 

Some explicit instruction would be much appreciated. So far I've been able to
1. Clone the repo locally
2. Create a folder with the question, answer, and SLO files
3. Commit the changes
4. ??

Pushing or Pull Requesting the changes is what I am unsure on how to do!

**Comment by agritter** [comment, marked as answer]
I know this is a little old, but here is how I added questions:

1. Fork this repo (there is a button at the top right)
2. Clone the forked repo you made (or press "." to make changes in your browser)
3. Make the changes you want to make
4. Commit and push those changes to your fork
5. Use the contribute button on your forked repo to make a pull request (see below)

![image](https://user-images.githubusercontent.com/44212308/220718111-7f60cc62-f03a-4413-ac6c-6b3ad60b88dd.png)
